compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -1327,6 +1374,7 @@ pub(crate) fn hir_crate_items(tcx: TyCtxt<'_>, _: ()) -> ModuleItems {
         trait_items: Vec<TraitItemId>,
         impl_items: Vec<ImplItemId>,
         foreign_items: Vec<ForeignItemId>,
+        body_owners: Vec<LocalDefId>,
     }
 
     impl<'hir> Visitor<'hir> for CrateCollector<'hir> {
```

Comment:
I wonder if we can merge this visitor and the one for `hir_module_items` using some flag to switch behaviour.